tribe-ext-divi-theme-compatibility.php

```diff
@@ -71,8 +71,22 @@ public function init() {
 				add_filter( 'parse_query', array( $this, 'remove_et_custom_posts_per_page' ), 100 );
 			}
 
-			add_action( 'wp_head', array( $this, 'fix_divi_widget_styles' ) );
-			add_action( 'wp_head', array( $this, 'fix_db_widget_styles' ) );
+			// Checking if Events Calendar PRO is active
+			if ( Tribe__Dependency::instance()->is_plugin_active( 'Tribe__Events__Pro__Main' ) ) {
+
+				// If Divi is the active / parent theme load the appropriate styles
+				if ( 'Divi' == get_template() ) {
+					add_action( 'wp_head', array( $this, 'fix_divi_widget_styles' ) );
+				}
+
+				// Needed for is_plugin_active()
+				require_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+				// If Divi Builder Plugin is active load the appropriate styles
+				if ( is_plugin_active( 'divi-builder/divi-builder.php' ) ) {
+					add_action( 'wp_head', array( $this, 'fix_db_widget_styles' ) );
+				}
+			} // if ( class_exists( 'Tribe__Events__Pro__Main', false ) )
```

Comment:
fix last comment to match actual code or just simply to something like `// if Events Calendar PRO is active`